sdks/java/io/iceberg/src/main/java/org/apache/beam/sdk/io/iceberg/IcebergIO.java

```diff
@@ -77,6 +90,21 @@ public WriteRows to(DynamicDestinations destinations) {
       return toBuilder().setDynamicDestinations(destinations).build();
     }
 
+    /**
+     * Sets the frequency at which data is committed and a new {@link org.apache.iceberg.Snapshot}
+     * is produced.
+     *
+     * <p>Every triggeringFrequency duration, all accumulated {@link
```

Comment:
Might want to soften the language so users know it is approximate, or we'll get complaints about it not lining up perfectly.

Reply:
good call, I softened it up